app/stores/NetworkStatusStore.js

```diff
@@ -25,9 +26,17 @@ export default class NetworkStatusStore extends Store {
   }
 
   @computed get syncPercentage(): number {
-    if (this.networkDifficulty > 0 && this._localDifficultyStartedWith != null) {
+    if (this.networkDifficulty > 0 && this._localDifficultyStartedWith !== null) {
```

Comment:
I know. `undefined` value should not be possible here. If it is I would handle that explicitly.